.travis.yml

```diff
@@ -1,19 +1,30 @@
 sudo: false
 language: php
 
-php:
-    - 5.4
-    - 5.5
-    - 5.6
-    - 7.0
-    - 7.1
+cache:
+    directories:
+        - $HOME/.composer/cache
 
-before_install:
-    - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then composer require satooshi/php-coveralls:^1.0 --no-update; fi;
+matrix:
+    include:
+        - php: 5.4
```

Comment:
Are you sure, that none of PHPUnit versions provided by Travis CI can't be used?

The matrix listing every possible PHP version does look a bit unusual.

Reply:
@aik099, thank you for your reply.

After finding the default `phpunit` execution provided by Travis, I update the PHP matrix lists.

You're right. Some PHP versions don't need the `phpunit` definition because it has been already existed the current version provided by Travis.